ui/src/components/App/AppCardStep.tsx

```diff
@@ -38,10 +38,10 @@ export default function AppCardStep({
       </Disclosure.Button>
       {expandedContent && isExpandable && (
         <Transition
-          enter="transition-opacity duration-300"
+          enter="transition-opacity duration-200"
           enterFrom="opacity-0"
           enterTo="opacity-100"
-          leave="transition-opacity duration-300"
+          leave="transition-opacity duration-200"
           leaveFrom="opacity-100"
           leaveTo="opacity-0"
         >
```

Comment:
**suggestion (ux)**: the animation feels a bit clunky to me:

https://github.com/inngest/inngest/assets/4291707/d4edbe83-4bc8-4c70-b2b7-6b432d660847

An animation in this style might fit better:

https://github.com/inngest/inngest/assets/4291707/b26b6d50-4bde-4edd-a58c-8d85fdc5fcac